README.md

```diff
@@ -20,10 +22,37 @@ range of applications using the [Google Maps Android API][android-site].
 
 <p align="center"><img width="90%" vspace="20" src="https://cloud.githubusercontent.com/assets/1950036/6629704/f57bc6d8-c908-11e4-815a-0d909fe02f99.gif"></p>
 
+## Requirements
+
+* Android API level 15+
+* Maps SDK via Google Play Services (this library is not yet compatible with the [standalone Maps SDK BETA library])
+
+## Installation
+
+```groovy
+dependencies {
+    implementation 'com.google.maps.android:android-maps-utils:0.6.2'
```

Comment:
Can this be auto-updated for new releases? Or should we have a documented release process somewhere?

It's easy to forget to increment this manually. We do have the Maven Central badge above the document title.

Reply:
I was thinking that the alternative is to write something like this: 

```groovy
dependencies {
    implementation 'com.google.maps.android:android-maps-utils:$latest-version'
}
```

...and then link the user to our changelog to get the latest version. From our standpoint, it's the easiest thing to do so that we don't have to update our `README.md` each time we release. However, from a ease-of-use standpoint, I think it's much better to update this each time so people can just copy/paste the above line.

I have a private doc for how we release - I'm happy to create a `RELEASING.md` doc describing this process so that we can update it each time. Lmk what you think.

Reply:
>However, from an ease-of-use standpoint, I think it's much better to update this each time so people can just copy/paste the above line.

I agree

>I'm happy to create a RELEASING.md doc describing this process so that we can update this version each time. Lmk what you think.

Sounds good to me! This also helps demystify the build/release process the project is using if others want to emulate it for their own projects.